bindings/CMakeLists.txt

```diff
@@ -1,3 +1,31 @@
+if(EARLY_BUILD)
+  add_executable(sizes sizes.cpp)
+  target_link_libraries(sizes PRIVATE ghexlib)
+  return()
+endif()
+
+message(STATUS "CONFIGURE sizes tool")
```

Comment:
Do you need this message now?

Reply:
no. Can remove it if you want.

Reply:
actually, it's not there in the new version.